diff --git a/ports/raspberrypi/boards/waveshare_rp2350_touch_lcd_2/mpboardconfig.mk b/ports/raspberrypi/boards/waveshare_rp2350_touch_lcd_2/mpboardconfig.mk
index 27f915a..f83ef95 100644
--- a/ports/raspberrypi/boards/waveshare_rp2350_touch_lcd_2/mpboardconfig.mk
+++ b/ports/raspberrypi/boards/waveshare_rp2350_touch_lcd_2/mpboardconfig.mk
@@ -6,4 +6,4 @@ USB_MANUFACTURER = "Waveshare Electronics"
 CHIP_VARIANT = RP2350
 CHIP_FAMILY = rp2
 
-EXTERNAL_FLASH_DEVICES = "W25Q32JVxQ"
\ No newline at end of file
+EXTERNAL_FLASH_DEVICES = "W25Q32JVxQ"
diff --git a/ports/raspberrypi/boards/waveshare_rp2350_touch_lcd_2/pins.c b/ports/raspberrypi/boards/waveshare_rp2350_touch_lcd_2/pins.c
index 2d91805..dd3fb3c 100644
--- a/ports/raspberrypi/boards/waveshare_rp2350_touch_lcd_2/pins.c
+++ b/ports/raspberrypi/boards/waveshare_rp2350_touch_lcd_2/pins.c
@@ -60,7 +60,7 @@ static const mp_rom_map_elem_t board_module_globals_table[] = {
     { MP_ROM_QSTR(MP_QSTR_LCD_RST), MP_ROM_PTR(&pin_GPIO20) },
     { MP_ROM_QSTR(MP_QSTR_DISPLAY_RST), MP_ROM_PTR(&pin_GPIO20) },
     { MP_ROM_QSTR(MP_QSTR_MISO), MP_ROM_PTR(&pin_GPIO20) }, // Default SPI
-    
+
     { MP_ROM_QSTR(MP_QSTR_GP21), MP_ROM_PTR(&pin_GPIO21) }, // Pin 24
     { MP_ROM_QSTR(MP_QSTR_GP22), MP_ROM_PTR(&pin_GPIO22) }, // Pin 25
     { MP_ROM_QSTR(MP_QSTR_GP23), MP_ROM_PTR(&pin_GPIO23) }, // Pin 22
